frontend/src/lib/components/ExportButton/exportsLogic.ts

```diff
@@ -96,13 +97,20 @@ export const exportsLogic = kea<exportsLogicType>([
         startExport: async ({ exportData }) => {
             if (isLocalExport(exportData.export_context)) {
                 try {
-                    downloadBlob(
-                        new Blob([exportData.export_context.localData], { type: exportData.export_context.mediaType }),
-                        exportData.export_context.filename
-                    )
+                    // check size directly
+                    const blob = new Blob([exportData.export_context.localData], {
+                        type: exportData.export_context.mediaType,
+                    })
+                    if (blob.size > MAX_EXPORT_SIZE_BYTES) {
```

Comment:
What makes you think this is the thing stopping exports from happening? There are reports that blobs support sizes of 500MB to 2GB - so im not so sure why we'd wanna limit these to 10MB?

Reply:
i'm not completely sure this is the problem if i'm being honest. i just got a report from a customer that the 300k row limit was wrong in their case. they had deeply nested JSON in the export body and were only able to export ~50k rows. i couldn't find where the 300k row limit is enforced so i went with this instead. but i'm removing it. for now we can just add timestamp to the body.